src/sql/parts/jobManagement/views/jobsView.component.ts

```diff
@@ -937,7 +940,12 @@ export class JobsViewComponent extends JobManagementView implements OnInit, OnDe
 			let alerts = this.jobAlerts[jobId];
 			job[0].alerts = alerts;
 		}
-		return job && job.length > 0 ? job[0] : undefined;
+		if (job && job.length > 0) {
```

Comment:
we're accessing `job[0]` above in the method.  Any reason we don't need to check `job.length > 0` in those cases above?